charmap.asm

```diff
@@ -34,7 +34,7 @@
 	charmap "<TM>",      $5c ; "TM"
 	charmap "<TRAINER>", $5d ; "TRAINER"
 	charmap "<ROCKET>",  $5e ; "ROCKET"
-	charmap "<DEXEND>",  $5f
+	charmap "<DEXEND>",  $5f ; unused
```

Comment:
Other chars are unused too, like `"<TRAINER>"` and `"<ROCKET>"`; I don't think they all need such comments.
```suggestion
	charmap "<DEXEND>",  $5f
```

Reply:
No specific issue with removing that comment, although I added that because most of the characters under "Actual characters" specifically list when they are unused. Regardless, you're right that it makes more sense to either apply it to all of the control characters or none of them, so that would belong in a separate pull request.